src/helpers/transitionProperty.js

```diff
@@ -0,0 +1,9 @@
+const transitionProperty = () => {
+	if (window.ontransitionend === undefined && window.onwebkittransitionend !== undefined) {
+		return 'WebkitTransition'
+	} else {
+		return 'transition'
+	}
```

Comment:
Looks like duplicate code. Code we reuse together with `transitionEnd` function?

Reply:
We could turn those two into one function that returns both? Or set a Webkit flag. Personally, I have a rule only to refactor and DRY for 3 or more occurrences of similar code so this one has flown under the radar 🤡